runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -510,45 +509,45 @@ private void applyUndoRecordUnsafe(SMREntry entry) {
     /**
      * Apply an SMR update to the object, possibly optimistically.
      *
-     * @param entry The entry to apply.
+     * @param record The SMR record to apply.
      */
-    private Object applyUpdateUnsafe(SMREntry entry) {
-        log.trace("Apply[{}] of {}@{} ({})", this, entry.getSMRMethod(),
-                Address.isAddress(entry.getGlobalAddress()) ? entry.getGlobalAddress() : "OPT",
-                entry.getSMRArguments());
+    public Object applyUpdateUnsafe(SMRRecord record) {
```

Comment:
why public?

Reply:
It was public before. Changed to private.